pulsar/resource_pulsar_namespace.go

```diff
@@ -646,3 +740,27 @@ func unmarshalPersistencePolicies(v *schema.Set) *utils.PersistencePolicies {
 
 	return &persPolicies
 }
+
+func unmarshalPermissionGrants(v []interface{}) ([]*types.PermissionGrant, error) {
+	var permissionGrants []*types.PermissionGrant
+	for _, grant := range v {
+		data := grant.(map[string]interface{})
+
+		var permissionGrant types.PermissionGrant
+		permissionGrant.Role = data["role"].(string)
+
+		var actions []common.AuthAction
+		for _, action := range data["actions"].(*schema.Set).List() {
+			authAction, err := common.ParseAuthAction(action.(string))
+			if err != nil {
+				return permissionGrants, fmt.Errorf("ERROR_INVALID_AUTH_ACTION: %w", err)
```

Comment:
i think it's customary to just return nil as the first param when returning an error as the second.